bundles/InstallBundle/Resources/install.sql

```diff
@@ -865,11 +865,17 @@ CREATE TABLE `quantityvalue_units` (
   `group` varchar(50) DEFAULT NULL,
   `abbreviation` varchar(20) NOT NULL,
   `longname` varchar(250) DEFAULT NULL,
-  `baseunit` varchar(10) DEFAULT NULL,
+  `baseunit` `baseunit` INT(11) UNSIGNED DEFAULT NULL,
```

Comment:
I guess this is not right

Reply:
Why not? The column baseunit contains the id of the base unit. If you do not set a base unit then it is null.

Reply:
Oh sorry, my fault. Is fixed now